README.md

```diff
@@ -34,6 +34,8 @@ rustup override set nightly
 ```sh
 # e.g. Ubuntu or Debian
 apt-get install llvm-6.0 libgc-dev
+ln -sf /usr/bin/llvm-config-6.0 /usr/bin/llvm-config
+apt-get install zlib1g-dev
```

Comment:
> `apt-get install zlib1g-dev`

This line should be merged into `apt-get install llvm-6.0 libgc-dev zlib1g-dev`.

> `ln -sf /usr/bin/llvm-config-6.0 /usr/bin/llvm-config`

I think this is an optional operation, so comment it out and add a note.
e.g. `# ln -sf /usr/bin/llvm-config-6.0 /usr/bin/llvm-config # May help if you encounter llvm-related error`